### ISSUE TITLE
Some doubts about Bidirectional Lstm

### ISSUE THREAD

**Issue by Ostnie** [kamalkraj/Named-Entity-Recognition-with-Bidirectional-LSTM-CNNs#3]
Hi, I'm doing some job about bidirectional lstm, and the programs are similiar with yours, but my data seems to have some problems, because lstm can only deal with the data whose dimention is more than 3D. In my data, the data size is (10000,128,10),and the label is (10000,10),it's a classfication problem, and my model is like this:

    model = Sequential()
    model.add(Bidirectional(LSTM(100, return_sequences=True),input_shape=(128, 10)))
    model.add((Dense(units=200,activation='tanh')))
    model.add((Dense(kind, activation='softmax')))

When I begin the traning, I got the error:
 Error when checking target: expected dense_2 to have 3 dimensions, but got array with shape (10000, 10), how can I change my label size ?could you please help me?

**Comment by Ostnie** [comment]
Oh,I find it can successfully running if I get rid of return_sequences=True,that means 

model.add(Bidirectional(LSTM(100),input_shape=(128, 10)))

but the acc is very low, I don't know whether it is correct to modify the programs

**Comment by kamalkraj** [comment]
if you put return_sequences=True then in the next layer you should use
timedistributed dense layer , it is available in keras as wrapper
